plugins/plugin-client-common/web/css/static/repl.scss

```diff
@@ -13,6 +13,9 @@
   /* streaming output of stdout; make sure stderr occurs below */
   flex-direction: column;
 }
+.repl-result.overflow-visible {
+  overflow: visible;
+}
```

Comment:
The css rule here makes the dropdown visible. I tried to wrap a div on the Dropdown element to break the scope, in which case I'll have to make the div `position:absolute` and take care of the spacing by myself. So, I chose to add the class `overflow-visible` to `repl-result` and make the overflow visible when the output model is `RadioTable`.